bin/feature

```diff
@@ -210,7 +215,7 @@ when 'merge'
       # rebase the unpushed master commits if any.
       "git rebase --preserve-merges origin/#{dev_branch}",
       # merge the feature branch into master
-      "git merge --no-ff --edit -m #{pull_info[:description].shellescape} \"#{feature}\"",
+      "git merge --no-ff --edit -m #{pull_desc.shellescape} \"#{feature}\" || #{abort_merge}",
```

Comment:
It _appears_ that `feature` needs to be shellescaped as well but this isn't anything new.